.azure/templates/test-distribution.yml

```diff
@@ -0,0 +1,33 @@
+jobs:
+- job: test_distribution
+  displayName: Test Distribution
+  variables:
+  - name: runCodesignValidationInjection
+    value: false
+  - name: skipComponentGovernanceDetection
+    value: true
+  pool:
+    vmImage: ubuntu-latest
+
+  steps:
+  - task: DownloadBuildArtifacts@0
+    displayName: Download Test
+    inputs:
+      artifactName: artifacts
+      itemPattern: artifacts/bin/linux/x64_Release_openssl/msquictest
+      downloadPath: $(Build.SourcesDirectory)
+  - task: DownloadBuildArtifacts@0
+    displayName: Download Package
+    inputs:
+      artifactName: distribution
+      itemPattern: distribution/*.deb
+      downloadPath: $(Build.SourcesDirectory)
+  - pwsh: |
+      sudo apt-add-repository ppa:lttng/stable-2.12
+      sudo apt-get update
+      sudo apt-get install -y lttng-tools
+      sudo dpkg -i $(Build.SourcesDirectory)/distribution/libmsquic_2.1.0_amd64.deb
+  - pwsh: |
+      echo $(Build.SourcesDirectory)/artifacts/bin/linux/x64_Release_openssl/msquictest
+      $(Build.SourcesDirectory)/artifacts/bin/linux/x64_Release_openssl/msquictest --gtest_filter=ParameterValidation.ValidateApi
+      echo Complete
```

Comment:
@wfurt, @ManickaP do you have any idea why this test wouldn't be working? I only get the following output in Azure Pipelines:
```
========================== Starting Command Output ===========================
/usr/bin/pwsh -NoLogo -NoProfile -NonInteractive -Command . '/home/vsts/work/_temp/2d25da3b-6e8a-460e-85cc-813d6f944367.ps1'
/home/vsts/work/1/s/artifacts/bin/linux/x64_Release_openssl/msquictest
Complete
Finishing: PowerShell
```
If it can't find libmsquic, I'd expect to get an error like this:
```
error while loading shared libraries: libmsquic.so.2: cannot open shared object file: No such file or directory
```
At least that's what happened locally. I went through these steps locally and it worked just fine. Ideas?

Reply:
It is exactly same OS as you build on ? you can try to add `ldd  artifacts/bin/linux/x64_Release_openssl/msquictest` (and possibly `ls -al artifacts/bin/linux/x64_Release_openssl`)

Reply:
@wfurt
```
/home/vsts/work/1/s/artifacts/bin/linux/x64_Release_openssl/msquictest
	linux-vdso.so.1 (0x00007ffce6f8e000)
	libmsquic.so.2 => /lib/x86_64-linux-gnu/libmsquic.so.2 (0x00007f899226c000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f8992266000)
	libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f8992084000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f8991f35000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f8991f1a000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f8991ef5000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f8991d03000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f89929a5000)
total 37628
drwxr-xr-x 2 vsts docker     4096 Apr 12 15:59 .
drwxr-xr-x 3 vsts docker     4096 Apr 12 15:59 ..
-rw-r--r-- 1 vsts docker 38521656 Apr 12 15:59 msquictest
Complete
```